shared/references/cce-sle12-avail.txt

```diff
@@ -1,31 +1,3 @@
-CCE-92254-2
-CCE-92255-9
-CCE-92256-7
-CCE-92257-5
-CCE-92258-3
-CCE-92259-1
-CCE-92260-9
-CCE-92261-7
-CCE-92262-5
-CCE-92263-3
-CCE-92264-1
-CCE-92265-8
-CCE-92266-6
-CCE-92267-4
-CCE-92268-2
-CCE-92269-0
-CCE-92270-8
-CCE-92271-6
-CCE-92272-4
-CCE-92273-2
-CCE-92274-0
-CCE-92275-7
-CCE-92276-5
-CCE-92277-3
-CCE-92278-1
-CCE-92279-9
-CCE-92280-7
-CCE-92281-5
```

Comment:
You are probably removing more CCEs than necessary in this PR. Could you check, please?